pipeline/alert.go

```diff
@@ -209,6 +206,11 @@ type AlertNode struct {
 	// An empty value indicates the level is invalid and is skipped.
 	Crit *ast.LambdaNode
 
+	// Filter expression for reseting the WARNING alert level to OK.
```

Comment:
Doesn't a reset set it to a different level not necessarily the OK state? Meaning a CRITICAL remains CRITICAL until the reset condition is false, then the level is determined again using the previous behavior. Which could be WARNING, INFO or OK.